app/views/catalog/_universal_viewer_default.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if resource.class.can_have_manifests? && resource.try(:member_ids).present? %>
   <div class="intrinsic-container intrinsic-container-16x9 uv-container">
-    <iframe src="/uv#?manifest=<%= request.base_url + main_app.polymorphic_path([main_app,:manifest,resource]) %>" allowfullscreen="true"></iframe>
+    <iframe src="/uv/uv#?manifest=<%= request.base_url + main_app.polymorphic_path([main_app,:manifest,resource]) %>" allowfullscreen="true"></iframe>
```

Comment:
Do we know why we need this additional `/uv` in the path? if so it would be nice to have a comment since it looks like a typo or something.

Reply:
Yeah, they just changed the path for the `uv.html` to be within the dist directory.

Reply:
I tried to change this back and it wouldn't work. :(